src/context.rs

```diff
@@ -383,6 +383,12 @@ impl<'ctx> Context<'ctx> {
         }
     }
 
+    /// Checks if a given type is supported by the build target.
+    #[cfg(feature="master")]
+    pub fn supports_target_dependent_type<'a>(&'a self, c_type: CType) -> bool {
+        self.get_target_info().supports_target_dependent_type(c_type.to_sys())
+    }
```

Comment:
Do you really need this method?
I would prefer avoiding this as this will alloc and free the TargetInfo immediately.

Reply:
Would you suggest to remove this method and change the `target_info` function a bit to something like this:
```
pub fn supports_target_dependent_type(&self, c_type: CType) -> bool
```